source/GuiElements.cpp

```diff
@@ -149,9 +149,14 @@ void WaypointDisplay::setManager(WidgetManager *manager) {
 	}
 }
 
+void PlayerControls::updateConnectedClients()
```

Comment:
This method might need a name change, it seems to me like it sets the selected client, not updates the connected clients...